shell/platform/windows/key_event_handler.cc

```diff
@@ -108,7 +110,9 @@ void KeyEventHandler::KeyboardHook(FlutterWindowsView* view,
   event.AddMember(kScanCodeKey, scancode, allocator);
   event.AddMember(kCharacterCodePointKey, character, allocator);
   event.AddMember(kKeyMapKey, kWindowsKeyMap, allocator);
+#ifndef WINUWP
   event.AddMember(kModifiersKey, GetModsForKeyState(), allocator);
```

Comment:
Not having this is going to cause some problems on the framework side: it will attempt to synchronize the list of modifiers that are down, and won't find any, so will assume they aren't down. Probably all shortcuts that use modifiers will stop working. You will probably have to keep track of the modifiers that are down yourself, or find another API in UWP that can tell you.

Reply:
Agreed, this should have a TODO.  Looks like the winrt equiv is possibly https://docs.microsoft.com/en-us/uwp/api/windows.ui.core.corewindow.getkeystate?view=winrt-19041

Reply:
I've added this to https://github.com/flutter/flutter/issues/70202

Reply:
Done in next commit